src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2020, 2024, Oracle and/or its affiliates. All rights reserved.
+// Copyright (c) 2020, 2025, Oracle and/or its affiliates. All rights reserved.
 // Copyright (c) 2020, 2023, Arm Limited. All rights reserved.
 // Copyright (c) 2020, 2022, Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
Instead of modifying the Oracle Copyright, we should add one for Rivos. That can be done in a future PR as well.

Reply:
Thanks for revewing!
I've added the copyright in https://github.com/openjdk/jdk/pull/23580.